src/harness/test_harness_objects.py

```diff
@@ -356,6 +349,56 @@ def getCbsdsWithAtLeastOneAuthorizedGrant(self):
         cbsd_objects.append(cbsd_object)
     return cbsd_objects
 
+  def _withMaximumBatchSize(self, requests, request_name):
+    """Split the requests into batches and call the given function.
+
+    Args:
+      requests: The requests to split by batchsize.
+      request_name: The name of the request. For logging.
+    Yields:
+      Lists of requests containing at most maximum_batch_size requests.
+    """
+    maximum_batch_size = self.testcase._sas.maximum_batch_size
+    num_batches = int(math.ceil(len(requests) / float(maximum_batch_size)))
+    logging.info('Sending %i %s requests in batches of %i (%i batches)',
+                 len(requests), request_name, maximum_batch_size, num_batches)
+    for iteration in range(0, num_batches):
+      batch_requests = requests[iteration * maximum_batch_size:(iteration + 1) *
+                                maximum_batch_size]
+      yield batch_requests
+
+  def _assertRegisteredWithMaximumBatchSize(self, registration_requests,
+                                            conditional_registration_data):
+    """Sends registration requests in batches up to the maximum batch size.
+
+    Args:
+      registration_requests: All registration requests we wish to send.
+      conditional_registration_data: Optional, any conditional registration data
+        for the CBSDs.
+    Returns:
+      The cbsd_ids from each registration request.
+    Raises:
+      AssertionError: If any registration responses are invalid.
+    """
+    # Make sure FccId and userId are injected before conditionals. The
+    # optimization in the SAS interface will prevent duplicate requests from
+    # being sent.
```

Comment:
the code looks good, this comment was not that clear to me(namely the text "in the SAS interface" is a bit confusing, because it is not the interface which forces for this optimization, but it is internally in the class "SasAdminImpl" in "sas.py" ),  so I think  this comment could be more clear.